vendor/src/github.com/docker/libnetwork/endpoint_cnt.go

```diff
@@ -108,6 +108,25 @@ func (ec *endpointCnt) EndpointCnt() uint64 {
 	return ec.Count
 }
 
+func (ec *endpointCnt) updateStore() error {
+retry:
```

Comment:
@calavera The only reason we get ErrKeyModified is if some other node writes to the same object since it was last written from this node. So doing a read again will give the updated dbIndex and the subsequent write will succeed. There is the reason for not having a backoff or timeout.